portal/views/teacher/teach.py

```diff
@@ -69,6 +69,75 @@
 
 @login_required(login_url=reverse_lazy('login_view'))
 @user_passes_test(logged_in_as_teacher, login_url=reverse_lazy('login_view'))
+def materials(request):
+
+    session_names = ["ks1_session_", "lks2_session_", "uks2_session_"]
+    resource_sheets_names = ["KS1_S", "LKS2_S", "UKS2_S"]
+    ks1_sessions = []
+    lks2_sessions = []
+    uks2_sessions = []
+    session_dictionaries = [ks1_sessions, lks2_sessions, uks2_sessions]
+    ks1_sheets = []
+    lks2_sheets = []
+    uks2_sheets = []
+    resource_sheets_dictionaries = [ks1_sheets, lks2_sheets, uks2_sheets]
+
+    for ks_index in range(0, len(session_names)):
```

Comment:
a better way would be to use enumerate - 
for ks_index, value in enumerate(session_names): ...